app/controllers/admin/submitters_controller.rb

```diff
@@ -42,5 +42,13 @@ class SubmittersController < Admin::ApplicationController
 
     # See https://administrate-prototype.herokuapp.com/customizing_controller_actions
     # for more information
+    def new
```

Comment:
Should we add test or two to the admin dashboard controller confirming the behavior in this controller action? (To clarify, that's not a rhetorical question; I'm not sure whether it make sense to test it or not.)

Reply:
yeah good call. I think a case for not passing the param and one passing it and showing it gets selected make sense.